### DISCUSSION TITLE
"There are outstanding issues with your GitHub account. Please contact GitHub Support for more information."

### DISCUSSION THREAD

**Discussion by muenyz** [community/community#65968] [GitHub Education]
### Hi everyone,

My approved submission has been expired.What happened?

**Comment by muenyz** [comment]
I was just approved on Aug 30

**Reply by zkdsaf** [reply]
你好，我也遇到同样的问题，请问你是怎么解决的

**Comment by lab596** [comment]
As the question suggests, the best course of action in this situation is to contact GitHub Support directly. They can provide specific information about the status of your submission and help you resolve any outstanding issues with your account. When reaching out be sure to provide them with all relevant details about your situation, including the approval date and any other information that may be pertinent. You can reach out via this link: https://support.github.com/

**Comment by muenyz** [comment]
我重新交了份申请又approved了，不知道之前什么情况给我取消了……&nbsp;


Muenyz
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Zhu ***@***.***&gt;; 
发送时间: 2023年9月6日(星期三) 上午9:52
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [community/community] &quot;There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.&quot; (Discussion #65968)





 
你好，我也遇到同样的问题，请问你是怎么解决的
